capstone/src/main/java/com/google/univiz/scorecard/ScorecardData.java

```diff
@@ -0,0 +1,91 @@
+package com.google.univiz.scorecard;
+
+import com.google.auto.value.AutoValue;
+import com.google.gson.annotations.SerializedName;
+import com.ryanharter.auto.value.gson.GenerateTypeAdapter;
+import javax.annotation.Nullable;
+
+/**
+ * ScorecardData is a class representing a single university/college/institution from the parsed
+ * JSON data directly from the CollegeScorecard API. The parameters come directly from the
+ * CollegeScorecard API Data Dictionary
+ * (https://collegescorecard.ed.gov/assets/CollegeScorecardDataDictionary.xlsx), which outlines the
```

Comment:
Just a heads up that there are other Javadoc constructs you can use for links - namely `<a href=""></a>`.

Just for future reference, no need to change anything now.